### ISSUE TITLE
High octane fuel, cryogenics and graphene.

### ISSUE THREAD

**Issue by bitwiseshiftleft** [Deadlock989/IndustrialRevolution#147]
Hi Deadlock, thanks for making this: it's awesome!

Currently high-octane fuel is unlocked at steel science, but it's stored in high-pressure canisters.  These are unlocked at cryogenics, which is also steel science.  But making them requires nanoglass, which is unlocked by graphene at chrome science.

Suggestion: move high-octane fuel tech to chrome science, and make it dependent on graphene tech.  Move canisters from cryogenics tech to graphene tech.  Graphene depends on cryogenics, so you don't need a second dependency.

Alternative suggestion: make high-octane fuel not go in high-pressure canisters.  If it's really made of ethanol and petrol, it can probably be used to fuel a train without using any nanoglass.

**Comment by Deadlock989** [comment]
Thanks for this - cryogenics tech is one of a small number of techs which is exempt from automatic tech tree dependency generation, so the nanoglass prerequisite is being missed. I will review this soon.

**Comment by Deadlock989** [comment]
I think the solution here is close to your first suggestion above, except that canisters and canister filling should actually constitute their own individual tech. Barrelling was moved out of vanilla's "fluid handling" bundle for similar reasons a long time ago as it was awkward to have barrelling and also pumps and tanks in the same tech - tech dependencies around canisters/nanoglass would benefit from a similar approach. 

I debated with myself during development whether high octane fuel should be a full liquid fuel and then used as barrelled fuel instead, which would be another way to resolve the graphene dependency. There are pros and cons but on balance the cons won.
